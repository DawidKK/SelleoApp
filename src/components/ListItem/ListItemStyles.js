import styled from 'styled-components';

const ListItemStyles = styled.li`
  background: ${props => props.theme.bgLight};
  box-shadow: ${props => props.theme.bs};
  color: ${props => props.theme.white};
  padding: 1rem 3rem;;
  font-weight: 300;
  font-size: 1em;
  margin-bottom: 1rem;
  margin-right: 1rem;
`;

export default ListItemStyles;
