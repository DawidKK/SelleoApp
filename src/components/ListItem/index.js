import React from 'react'
import ListItemStyles from './ListItemStyles'
import ErrorBoundry from '../ErrorBoundry'

export const ListItem = ({ children }) => {
  return (
    <ErrorBoundry>
      <ListItemStyles>
        {children}
      </ListItemStyles>
    </ErrorBoundry>
  )
}

export default ListItem
