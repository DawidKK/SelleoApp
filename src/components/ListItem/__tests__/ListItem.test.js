import React from 'react';
import { shallow } from 'enzyme';
import ListItem from '..';

describe('MyComponent', () => {
  it('renders', () => {
    const component = shallow(<ListItem children="elo" />);
    expect(component).toMatchSnapshot();
  });
});
