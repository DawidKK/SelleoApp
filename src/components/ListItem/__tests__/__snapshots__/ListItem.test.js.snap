// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent renders 1`] = `
<ErrorBoundary>
  <styled.li>
    elo
  </styled.li>
</ErrorBoundary>
`;
